src/platform/Infineon/crypto/trustm/CHIPCryptoPALHsm_P256_trustm.cpp

```diff
@@ -136,11 +123,20 @@ CHIP_ERROR P256Keypair::Initialize(ECPKeyTarget key_target)
     }
     else
     {
+#if !ENABLE_TRUSTM_NOC_KEYGEN
```

Comment:
If you're having both if and else branches anyway, having the test not be negated would make the code easier to read.

Reply:
Will make it more readable in the next PR. Thanks for the input.